src/database/models/bookingReview.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const bookingReview = sequelize.define('BookingReviews', {
+    BookingId: DataTypes.INTEGER,
+    feedback: DataTypes.STRING,
+    rating: DataTypes.INTEGER
+  }, {});
+  bookingReview.associate = function(models) {
+     bookingReview.belongsTo(models.Bookings, {
+    foreignKey: {
+      allowNull: false,
+    },
+    });
+  };
+  return bookingReview;
+};
```

Comment:
@lemoissonM  great job but you forgot to add a new line at the end of the document

Reply:
Is it really necessary in models??